src/Nethermind/Nethermind.Specs/ReleaseSpec.cs

```diff
@@ -55,7 +55,12 @@ public class ReleaseSpec : IReleaseSpec
         // used only in testing
         public ReleaseSpec Clone() => (ReleaseSpec)MemberwiseClone();
 
-        public bool IsEip1559Enabled { get; set; }
+        public bool IsEip1559Enabled
+        {
+            get => _isEip1559Enabled || IsEip4844Enabled;
+            set => _isEip1559Enabled = value;
+        }
```

Comment:
NOTE: we might want to revisit this later since EIPs can require other EIPs.

This approach is kind of fragile, for example, EIP 4844 requires 2930 but we're not overriding `IsEip2930Enabled` like we're doing with `IsEip1559Enabled`.

Reply:
Yeah to be honest we should make sure that our release spec is following the spec, so have those cascades.